src/zjs_common.h

```diff
@@ -33,39 +33,40 @@
  */
 char *zjs_shorten_filepath(char *filepath);
 
-#ifdef DEBUG_BUILD
+// enable to use asserts even in release mode, e.g. for terse debugging
+#if 0
+#define USE_ASSERT
+#endif
```

Comment:
Wouldn't this be better to add as a make flag rather than requiring editing the header file?

Reply:
Yeah, it would be really nice if you could specify a comma separated list of all the different debug options, and/or 'all' for all of them.

Reply:
You can feel free to submit such a patch. :)

I don't think it makes much of a difference actually, this is very easy. It's on by default in DEBUG anyway, usually you would not want this on in release build.

But FYI for FTRACE I very specifically designed it the way I did so that you can turn on function tracing per-file by making the define in that file, instead of enabling it for the whole project and get way too much output. Yes there are fancier ways to handle it, again - feel free. This is quick and good enough for what I need.